exporter/loadbalancingexporter/loadbalancer.go

```diff
@@ -52,7 +52,20 @@ type loadBalancerImp struct {
 func newLoadBalancer(params exporter.CreateSettings, cfg component.Config, factory componentFactory) (*loadBalancerImp, error) {
 	oCfg := cfg.(*Config)
 
-	if oCfg.Resolver.DNS != nil && oCfg.Resolver.Static != nil {
+	var count = 0
```

Comment:
This works, and I like that it provides a short-circuit feature to the function. But I propose two alternatives:

1) externalize this into a `func checkMultipleResolversProvided(Config) error`, and call this function here. It should make your code easier to test and leave the existing function more readable.
2) alternatively, you can create a boolean `configured`, and set it to true whenever a resolver is not nil. On the next resolver, you check if configured is true and return the errMultipleResolversProvided error. You won't have the fail-fast behavior, though.